package.json

```diff
@@ -12,6 +12,7 @@
     "cross-env": "^5.0.1",
     "express": "^4.15.3",
     "express-handlebars": "^3.0.0",
+    "form-data": "^2.2.0",
```

Comment:
Is this used somewhere? I don't see any other references or `requires()` for this module.